#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_add_wrappers.dpatch by Daniel Kahn Gillmor <dkg@fifthhorseman.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enabling TCP wrapper support

@DPATCH@

--- redir-2.2.1.orig/Makefile
+++ redir-2.2.1/Makefile
@@ -9,8 +9,8 @@
 # if you would like support for TCP wrappers (and have libwrap.a
 # installed), remove these comments.
 
-WRAP_CFLAGS = # -DUSE_TCP_WRAPPERS 
-WRAP_LIBS = # -lwrap
+WRAP_CFLAGS = -DUSE_TCP_WRAPPERS 
+WRAP_LIBS = -lwrap
 
 # if your system needs any additional libraries (solaris, for example, 
 # needs the ones commented out below), edit this line.

